cpp/lib/ome/xml/meta/Convert.cpp

```diff
@@ -92,7 +92,6 @@ namespace
       convertFileAnnotations();
       convertListAnnotations();
       convertLongAnnotations();
-      convertMapAnnotations();
```

Comment:
The commits will be reverted after the 5.1 release so we can update to the new schema.  It's waiting on units to be done.